etc/ci/lockfile_changed.sh

```diff
@@ -8,6 +8,6 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-diff="$(git diff -- */*/Cargo.lock)"
+diff="$(find . -name 'Cargo.lock' | xargs git diff)"
```

Comment:
Use the `-print0` argument to `find` and the `-0` argument to `xargs` for robustness.